homeassistant/components/lektrico/strings.json

```diff
@@ -22,6 +22,38 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "state_e_activated": {
+        "name": "State e activated"
```

Comment:
What's state e?

Reply:
"State e activated" is called now  "Ev error". 
Ev = electric vehicle